docs/user/security/securing-communications/index.asciidoc

```diff
@@ -4,127 +4,178 @@
-data-in-transit. Browsers send traffic to {kib} and {kib} sends traffic to {es}.
-These communications are configured separately.
+Browsers send traffic to {kib} and {kib} sends traffic to {es}. These communications are configured separately to support Transport Layer
+Security (TLS/SSL) encryption for data-in-transit.
+
+TLS requires X.509 certificates to perform encryption and authentication of the application that is being communicated with. {kib} supports
+certificates and private keys in PEM or PKCS#12 format.
 
 [[configuring-tls-browser-kib]]
 ==== Encrypting traffic between the browser and {kib}
 
-NOTE: You do not need to enable {security-features} for this type of encryption. 
-
-. Obtain a server certificate and private key for {kib}.
+. *Obtain an X.509 certificate and corresponding private key for the {kib} server.*
 +
 --
-{kib} supports certificates/keys in both PKCS #12 key stores and PEM format.
-
-When you obtain a certificate, you must do at least one of the following:
-
-.. Set the certificate's `subjectAltName` to the hostname, fully-qualified domain name (FQDN), or IP address of the {kib} server.
+When you obtain a certificate, you must set its subject alternative name (SAN) correctly to ensure that modern web browsers with hostname
+verification will trust it. You can set the SAN to the {kib} server's fully-qualified domain name (FQDN), hostname, or IP address.
 
-.. Set the certificate's Common Name (CN) to the {kib} server's hostname or FQDN. Using the server's IP address as the CN does not work.
-
-You may choose to generate a certificate and private key using {ref}/certutil.html[the {es} certutil tool]. If you already used certutil to
-generate a certificate authority (CA), you would generate a certificate/key for Kibana like so (using the `--dns` param to set the
-`subjectAltName`):
+You may choose to generate an X.509 certificate signing request (CSR) and private key using the
+{ref}/certutil.html[`elasticsearch-certutil`] tool. For example:
 
 [source,sh]
 --------------------------------------------------------------------------------
-bin/elasticsearch-certutil cert --ca elastic-stack-ca.p12 --name kibana --dns localhost
+bin/elasticsearch-certutil csr --name kibana-server --dns <your_kibana_host>.com
 --------------------------------------------------------------------------------
 
-This will generate a certificate and private key in a PKCS #12 keystore named `kibana.p12`.
+This will produce a ZIP archive named `kibana-server.zip`. Extract that archive to obtain the PEM-formatted CSR (`kibana-server.csr`) and
+unencrypted private key (`kibana-server.key`). In this example, the CSR has a CN of `"kibana-server"` and a SAN of
+`"<your_kibana_host>.com"`.
 
+NOTE: you will need to send the CSR to a certificate authority (CA) to obtain a signed certificate. The certificate's signature will be
+verified by web browsers that are configured to trust the CA. However, be sure not to expose the private key when doing so.
```

Comment:
I changed this section to tell users to generate a CSR per @jkakavas's suggestion.